packages/dates/README.md

```diff
@@ -30,6 +30,23 @@ const timeZone2 = 'America/Toronto';
 const newDate = applyTimeZoneOffset(date, timeZone1, timeZone2);
 ```
 
+### `formatDate`
+
+Takes in a date object and two additional parameters, the locale and an optional options object. Returns a new date string with the applied locale and options.
+
+```ts
+import {formatDate} from '@shopify/dates';
+
+const date = new Date('2020-02-18Z14:00');
+const locales = 'en';
+const options = {
+  timeZone: 'America/New_York',
+  hour: 'numeric',
+};
+
+const newDate = formatDate(date, locales, options); // 9 AM
```

Comment:
The new date is just `9 AM`?

Reply:
Yeah, it's based on the options you pass it apparently. By doing `hour: 'numeric'` it'll just return the hour. If you don't specify hour it'll give you the date like this: `2/18/2020`